### DISCUSSION TITLE
Is socat always a mandatory dependency when installing via yum past RabbitMQ 3.6.2

### DISCUSSION THREAD

**Discussion by victorkeanework** [rabbitmq/chef-cookbook#621] [Q&A] [answered]
This is the impression I got after reading the documentation - https://www.rabbitmq.com/docs/install-rpm#package-dependencies

However I've stumbled across a deployment of RabbitMQ 3.8.9 that was installed via yum that contains no Socat RPM and was wondering how/if this is possible?

**Comment by lukebakken** [comment, marked as answer]
`socat` was removed as a dependency at some point, I don't remember when.

**Reply by victorkeanework** [reply]
It is still marked as a dependency in the official documentation - https://www.rabbitmq.com/docs/install-rpm#package-dependencies

**Reply by michaelklishin** [reply]
This is open source software. When in doubt, [consult the source, including `rabbitmq-server.spec`](https://github.com/rabbitmq/rabbitmq-packaging/blob/main/RPMS/Fedora/rabbitmq-server.spec#L17-L26).

**Reply by michaelklishin** [reply]
`socat` was removed in two steps in 2021 https://github.com/rabbitmq/rabbitmq-packaging/issues/8.
